src/Microsoft.AspNetCore.Razor.Language/Extensions/DefaultTagHelperOptimizationPass.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Razor.Language.Intermediate;
+
+namespace Microsoft.AspNetCore.Razor.Language.Extensions
+{
+    internal class DefaultTagHelperOptimizationPass : RazorIRPassBase, IRazorIROptimizationPass
+    {
+        // Run later than default order so other passes have a chance to modify the
+        // tag helper nodes.
+        public override int Order => 1000;
```

Comment:
yeah I should be using the constant here. The default for user code is 0. The default for our code is 1000 where we want to run **after** user code. This is one of those cases